docs/features/sharepoint.md

```diff
@@ -0,0 +1,81 @@
+# Sharepoint File Ingestion
+
+## Overview
+
+A new logic app has been created to ingest sharepoint documents into a blob container for Information Assistant to process. It starts with the folder provided and crawls down through each folder in the sharepoint site. This process runs once every 24 hours.
+
+### Restrictions
+
+#### No RBAC
+In order to access your files in Sharepoint, you will have to login via Entra with the user that has access to these sharepoint files. We recommend creating a new user to do this.
+
+#### Document Support
+This ingestion process is only for **documents supported by our current pipeline**, the current officially supported file types are; pdf, docx, html, htm, csv, md, pptx, txt, json, xlsx, xml, eml, msg. This ingestion process does **NOT** include lists or information found on pages in your sharepoint site. It is only for documents.
```

Comment:
Rather than re-list the file types here, simply point to the existing documentation for supported file types so we only have to maintain it in one place..